src/main/java/com/endava/soa_5/OOP/Human.java

```diff
@@ -0,0 +1,52 @@
+package com.endava.soa_5.OOP;
+
+public class Human extends Mammal implements Ability {
+
+    private int old;
```

Comment:
Did you consider age here?

Reply:
It was moved into Mammal class, cause all Mammals child classes can use this parameter.